stdlib/Pkg/src/REPLMode.jl

```diff
@@ -774,6 +782,15 @@ function do_resolve!(ctx::Context, tokens::Vector{Token})
     API.resolve(ctx)
 end
 
+function do_activate!(ctx::Context, tokens::Vector{Token})
+    !isempty(tokens) && cmderror("`activate` does not take any arguments")
+    API.activate()
+end
```

Comment:
Shouldn't we allow one token, for the single-argument version: `activate(path::String)`?